rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    book.rental << self
```

Comment:
- [ ] What you defined in your Book class is `rentals`. To avoid errors kindly fix the syntax error.